### DISCUSSION TITLE
Rename Toggle for clarity

### DISCUSSION THREAD

**Discussion by MonkeyBars3k** [MonkeyBars3k/ReaScripts#55] [Ideas]
"Smart Glue/Unglue contextual" something appears to be contender

**Comment by basicchannels** [comment]
This is a tricky one for me, because toggle actually does describe what it's doing best in a way. 

Problem with smart glue is, it isn't smart enough, ie with that name you'd expect it to know Not to try and glue again if nothing has been changed or added, and that if you are 3 levels in to nesting, you're much more likely to want to unglue all the way back iteratively. Or if you make an edit halfway through, ie in the second nested lot, and hit toggle:glue again then it would guess that you want to glue it, rather than unglue one further step back because that would make less sense. 

Perhaps something like Quick Glue or something or Glue Preview

**Reply by MonkeyBars3k** [reply]
That seems an unnecessarily high bar for smart.￼

**Comment by MonkeyBars3k** [comment]
"Toggle" is only kind of accurate as it nearly always refers to changing a Boolean, whereas GR is a series of complex actions really, not just Glue/Unglue. So I do think the name change is warranted.

**Comment by MonkeyBars3k** [comment]
It comes back to our discussions before I started dev:
- Open/close
- Edit/Save/Propagate
- etc.

**Comment by MonkeyBars3k** [comment]
https://github.com/MonkeyBars3k/ReaScripts/issues/59
